birdhouse/env.local.example

```diff
@@ -206,7 +206,9 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 # allow jupyterhub user selection of which notebook image to run
 # see https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html
 #export ENABLE_JUPYTERHUB_MULTI_NOTEBOOKS="
-#c.DockerSpawner.image_whitelist = {'pavics': os.environ['DOCKER_NOTEBOOK_IMAGE'],
+#c.DockerSpawner.image_whitelist = {'pavics': os.environ['PAVICS_E2E_NOTEBOOK_IMAGE'],
+#                                   'eo-crim': os.environ['EO_NOTEBOOK_IMAGE'],
+#                                   'nlp-crim': os.environ['NLP_NOTEBOOK_IMAGE'],
```

Comment:
For test infra on a real jupyter instance, you'll need to change the test `env.local` in DACCS/daccs-configs repo, in order to see these images.